gaitmap/parameters/_spatial_parameters.py

```diff
@@ -93,6 +94,9 @@ class SpatialParameterCalculation(BaseSpatialParameterCalculation):
     Arc Length
         The overall arc length is directly calculated from the position of the sensor by adding the absolute changes in
         position at every time point.
+    Max. orientation change
+        The maximum change of angle in the sagittal plane. It is similar to the range of motion, however, the measured
+        parameter is value effected by other joints such as knee and hip as well.
```

Comment:
Might be a grammer error here.

Reply:
Took care of it, as I had to do some other gaitmap stuff as well and want to get a release done today :)

Thanks for your work on this!